tests/json/expected/cycle.json

```diff
@@ -7,5 +7,6 @@
     },
     "creation_date": "2022-03-31T21:50:52.349698",
     "start_date": "2022-03-31T00:00:00",
-    "end_date": "2022-03-31T23:59:59.999999"
+    "end_date": "2022-03-31T23:59:59.999999",
+    "version": "1.0"
```

Comment:
```suggestion
```

I believe cycle does not have a version attached. Right?

Reply:
👍